src/components/EntityInfo.vue

```diff
@@ -54,34 +54,32 @@
                     </span>
                   </template>
                   <template v-if="isPendingDissolution">
-                    You cannot view or change company information while the company is pending dissolution.
+                    You cannot view or change business information while the business is pending dissolution.
                   </template>
                   <template v-if="isNotInCompliance">
-                    You cannot view or change company information while the company is not in compliance.
+                    You cannot view or change business information while the business is not in compliance.
                   </template>
                 </v-tooltip>
               </span>
 
               <!-- Dissolve Company -->
               <span v-if="isAllowed(AllowableActions.DISSOLVE_COMPANY)">
+                <v-tooltip top content-class="top-tooltip" transition="fade-transition" nudge-right="7">
+                    <template v-slot:activator="{ on }">
                 <v-btn
                   small text color="primary"
                   id="dissolution-button"
                   :disabled="hasBlocker"
                   @click="promptDissolve()"
+                   v-on="on"
                 >
                   <img src="@/assets/images/Dissolution_Header_Icon.svg" alt="" class="pa-1">
                   <span class="font-13 ml-1">Dissolve this Business</span>
-                  <v-tooltip top content-class="top-tooltip" transition="fade-transition" nudge-right="7">
-                    <template v-slot:activator="{ on }">
-                      <span class="pl-1" v-on="on">
-                        <v-icon size="1rem">mdi-information-outline</v-icon>
-                      </span>
-                    </template>
-                    Dissolving the {{ entityTitle }} will make this {{ entityTitle }} historical
+                </v-btn>
```

Comment:
Now we don't need info icon to dissolve the business, tooltip will appear on hover of text or main icon

Reply:
"all types of entities"

(A corp is a type of entity. A coop is another. Firm, etc.)

Reply:
yes, for all entity